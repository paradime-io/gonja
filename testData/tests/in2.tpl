{% macro is_not_in(ls, value, if_true, if_false) -%}
    {%- if value|lower not in ls -%}
        {{- if_true -}}
    {%- else %}
        {{- if_false -}}
    {%- endif %}
{%- endmacro -%}


# contains

result: {{ is_not_in(["a","b"], "A", "does not contain", "contains") }}

# does not contain

result: {{ is_not_in(["a","b"], "C", "does not contain", "contains") }}