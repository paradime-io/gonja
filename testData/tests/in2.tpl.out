# contains

result: contains

# does not contain

result: does not contain